kolibri/plugins/coach/assets/src/views/common/ReportsControls.vue

```diff
@@ -74,19 +74,33 @@
         default: false,
       },
     },
+    data() {
+      return {
+        userSet: new Set(),
```

Comment:
We should be a little careful with this, as VueJS < 3 does not support reactive updates of ES6 Set objects: https://github.com/vuejs/vue/issues/2410

I _think_ in this case, the fact that we're always replacing the entire object is probably sufficient for this to retrigger a reactive update in the computed prop, but please be cautious in general with using Set and Map. This is the reason that we often just us a plain JS object when a Set or a Map might be more performant/semantic.